lotus-soup/testkit/role_miner.go

```diff
@@ -45,6 +50,8 @@ type LotusMiner struct {
 	*LotusNode
 
 	t *TestEnvironment
+
+	minerInstance *miner.Miner
```

Comment:
I think this could be called just `miner`. However it is important to note that this is just the `block producing` subsystem, and not the `MinerNode` - so I suggest a few renames further down.